Sources/Danger/BitBucketCloud.swift

```diff
@@ -1,228 +1,246 @@
 import Foundation
 
+// swiftlint:disable nesting
+
 public struct BitBucketCloud: Decodable {
     /// The activities such as OPENING, CLOSING, MERGING or UPDATING a pull request
-    public let activities: [BitBucketCloudActivity]
+    public let activities: [Activity]
 
     /// The comments on the pull request
-    public let comments: [BitBucketCloudComment]
+    public let comments: [Comment]
 
     /// The commits associated with the pull request
-    public let commits: [BitBucketCloudCommit]
+    public let commits: [Commit]
 
     /// The pull request and repository metadata
     public let metadata: BitBucketMetadata
 
     /// The PR metadata
-    public let pr: BitBucketCloudPR // swiftlint:disable:this identifier_name
+    public let pr: PR // swiftlint:disable:this identifier_name
 }
 
-public struct BitBucketCloudPR: Decodable {
-    public enum State: String, Decodable {
-        case declined = "DECLINED"
-        case merged = "MERGED"
-        case open = "OPEN"
-        case suspended = "SUPERSEDED"
-    }
-
-    private enum CodingKeys: String, CodingKey {
-        case author
-        case createdOn = "created_on"
-        case description
-        case destination
-        case id // swiftlint:disable:this identifier_name
-        case participants
-        case reviewers
-        case source
-        case state
-        case title
-        case updatedOn = "updated_on"
+extension BitBucketCloud {
+    public struct PR: Decodable {
```

Comment:
<!--
  0 failure: 
  1 warning:  Type name should ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 23;
-->

- :warning: Type name should be between 3 and 40 characters long: 'PR' (`type_name`)